### DISCUSSION TITLE
Aliases for target with different args

### DISCUSSION THREAD

**Discussion by ChugunovRoman** [microsoft/vscode-cmake-tools#2906] [General]
Hi, is it possible create aliases for one target for launch with different arguments?
I wanna launch an app from VSCode from status bar with different arguments.
I mean not just pass arguments, but it's look like a multiple targets

**Comment by bobbrow** [comment]
The status bar has limited functionality and may not be able to do what you want.  We have a setting `cmake.debugConfig` where you can set `"args"` for the target selected on the status bar, but if you want to specify different args for different targets, you'll need to use `launch.json` instead.  Some information about that can be found here: https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/debug-launch.md.  We also have some documentation about using `tasks.json` if you want to set up build dependencies too: https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/tasks.md
